lit_tests/kernel/wave/barriers.py

```diff
@@ -0,0 +1,272 @@
+# RUN: python %s | FileCheck %s
+
+import logging
+from typing import Callable
+import unittest
+import shark_turbine.kernel as tk
+import shark_turbine.kernel.lang as tkl
+import shark_turbine.kernel.wave as tkw
+from shark_turbine.kernel.wave.promotion import promote_node, promote_placeholders
+from shark_turbine.kernel.wave.barriers import add_shared_memory_barriers
+from shark_turbine.kernel.wave.hoisting import hoist_allocs
+from shark_turbine.kernel.wave.expansion import expand_graph
+from shark_turbine.kernel.lang.global_symbols import *
+from shark_turbine.kernel._support.tracing import CapturedTrace
+from shark_turbine.kernel._support.indexing import IndexingContext
+from shark_turbine.kernel.ops.wave_ops import *
+
+
+def run(func: Callable[[], None]) -> Callable[[], None]:
```

Comment:
We should probably move this to some common place, as it copypasted in every test.

Reply:
Good point, let me do that cleanup in a subsequent PR.